README.md

```diff
@@ -1,18 +1,15 @@
-# Positive Work Environment Task Force
-
+# Positive Work Environment Community Group
 
 W3C is a global community where participants choose to work together. In that community, we experience differences in language, location, nationality, and experience. In such a diverse environment, misunderstandings and disagreements happen, and in most cases can be resolved informally.
 
 This is the GitHub repository for the [Positive Work Environment Community Group](https://www.w3.org/community/pwe/).
 
 The CG picks up the work of the former [PWE Task Force](https://www.w3.org/Consortium/pwe/). 
 
-Chairs: Tzviya Siegman, Angel Li
+Chairs: Tzviya Siegman, Wendy Reid, Liz Lutgendorff
 
-The document can also be read directly on the [github.io page](https://w3c.github.io/PWETF/)
+This group maintains the W3C Code of Ethics and Professional Conduct, the draft version of the document can be read directly at [https://w3c.github.io/PWETF/]](https://w3c.github.io/PWETF/).
```

Comment:
s/Code of Ethics and Professional Conduct/Code of Professional Conduct/g
S/CEPC/CPC/g
per the decision in our last meeting, to rename the document.